Common/CPUDetect.cpp

```diff
@@ -273,16 +279,23 @@ std::string CPUInfo::Summarize()
 		sum = StringFromFormat("%s, %d cores", cpu_string, num_cores);
 		if (HTT) sum += StringFromFormat(" (%i logical threads per physical core)", logical_cpu_count);
 	}
+	// Intel and AMD have some different , some extension Intel or AMD don't have it
+	if (bMMX) sum += ", MMX";
```

Comment:
Personally, I think it's not very useful to report MMX.  PPSSPP requires SSE2, afaik, so it should be guaranteed that there's always MMX.  Not including this will also have a smaller impact to reporting.

-[Unknown]

Reply:
Yes , i will fix it

Reply:
You can check it again , i was changed the code